src/peer.rs

```diff
@@ -126,12 +126,21 @@ pub(crate) fn new_peer<T: HandshakeHandler>(
         // SPAWN WRITING THREAD
         //TODO: Bound
         let mut write_endpoint = endpoint.clone();
+        let write_active_connections = active_connections.clone();
+        let write_peer_id = peer_id.clone();
         // https://github.com/crossbeam-rs/crossbeam/issues/288
         let write_thread_handle = std::thread::spawn(move || loop {
```

Comment:
std:: thread::spawn is already import at line 4. If you want to prefix you can remove from the top

Reply:
I prefer to always prefix it, I think it's much more clear, I fixed the double import